docs/portal/sharing-permissions.md

```diff
@@ -79,23 +79,23 @@ When you manually create your sharing relationships using the search feature, yo
 
 ## Steps for Granting Sharing Permission
 
-Enabling sharing permissions includes the following steps.
-
 :::note
 As well as granting sharing permission in the UID2 Portal, you'll also need to integrate an SDK, or Snowflake functionality, into your code. See [Tokenized Sharing Overview](../sharing/sharing-tokenized-overview.md).
 :::
 
+Enabling sharing permissions includes the following steps.
+
 1. Log in to your UID2 Portal account.
 1. Click **Sharing Permissions**.
 1. Do one of the following:
 
-   - **Bulk Add Permission to Sharing Categories**: You can share with all current and future participants in one or more specific categories that you configure (Publishers, Advertisers, DSPs, or Data Providers). 
+   - **Add Permissions&#8212;Bulk**: You can share with all current and future participants in one or more specific categories that you configure (Publishers, Advertisers, DSPs, or Data Providers). 
 
    - **Review and Accept Recommendations**: Review the recommendations, add or clear recommendation categories as needed, and then click **Add Permissions**.
 
      If you accept a participant category, such as Advertisers or DSPs, sharing is enabled for current participants of that type, and also future participants of that type that join the UID2 ecosystem.
    
-   - **Add Permissions**: If you prefer, you can search for individual participants to share with. For details, see [Using Search to Add Sharing Relationships](#using-search-to-add-sharing-relationships).
+   - **Add Permissions&#8212;Individual**: If you prefer, you can search for individual participants to share with. For details, see [Using Search to Add Sharing Relationships](#using-search-to-add-sharing-relationships).
```

Comment:
You might want to add a section about Deleting sharing permissions.  You can delete individuals you've added through individual sharing permissions, but cannot delete individuals if you added them in bulk permissions.

Reply:
You can delete a whole group from Bulk permissions

Reply:
TYSM. Added this.